src/Core/Settings.h

```diff
@@ -726,6 +726,9 @@ class IColumn;
 #define MAKE_OBSOLETE(M, TYPE, NAME, DEFAULT) \
     M(TYPE, NAME, DEFAULT, "Obsolete setting, does nothing.", BaseSettingsHelpers::Flags::OBSOLETE)
 
+#define MAKE_OBSOLETE2(M, TYPE, NAME, DEFAULT) \
```

Comment:
Maybe this will be better?

```suggestion
#define MAKE_OBSOLETE_SERVER(M, TYPE, NAME, DEFAULT) \
```

Or even

```suggestion
#define MAKE_OBSOLETE_STRICT(M, TYPE, NAME, DEFAULT) \
```

And since now this settings indeed ignored, it is better to **refuse to start the server** if some of them still defined.

Thoughts?